server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreRestoreIT.java

```diff
@@ -256,7 +263,7 @@ private void testRestoreFlowMultipleIndices(int numberOfIterations, boolean invo
         ensureGreen(indices);
         for (String index : indices) {
             assertEquals(shardCount, getNumShards(index).totalNumShards);
```

Comment:
Same as above, should we be asserting:
```java
assertEquals(shardCount * 2, getNumShards(index).totalNumShards);
```
?

I think we need to check and change this for the tests with > 0 replicas